hack/test

```diff
@@ -105,7 +105,22 @@ function upgrade_pass {
 		--new-image=$UPGRADE_TO
 }
 
+function start_azurite {
+	if ! command -v docker >/dev/null; then
+		echo "docker not installed"
+	fi
+
+	# find running azurite.
+	RUNNING_AZURITE_CONTAINER=$(docker ps -q -f ancestor=arafato/azurite) 
```

Comment:
should now be `ancestor=quay.io/vdice/azurite`

Reply:
that's right.

Reply:
`ancestor =quay.io/vdice/azurite:git-9236ba3` works for me but not `ancestor=quay.io/vdice/azurite`